cechmate/filtrations/custom.py

```diff
@@ -23,3 +23,11 @@ def build(self, simplices):
         """
 
         self.simplices_ = simplices
+
+    def diagrams(self, simplices=None, show_inf=False, verbose=True, simplicial=True):
+        """
+        Redefine the Custom version of diagrams so that simplicial defaults to True, rather
```

Comment:
Could you help me understand why this one should default to true while the others default to false?

Reply:
My thought was that in all of the others, the simplicial complex is being generated in a way that we know it will be a simplicial complex. I thought it would be faster to pass ``False`` because we don't need to check it. My thought was not that the other ones are nonsimplicial, it's just that we don't need to *assert* that they are. For this one, we are generating the complex in a way that may not be simplicial. I assumed that most users would want the default behavior to be a simplicial complex, and they could choose to make something more subtle by passing ``simplicial=False``